playbooks/openshift/deploy_aosi.yml

```diff
@@ -0,0 +1,18 @@
+# #### Dependencies
+#
+# - setup.yml
+# - repo_from_source.yml
+#
+# #### Install ose from built rpms
+# ## defined in install_ose.yml ##
+#
+# - drop yum repo config onto all nodes
+# - yum install packages
+# - configure and run atomic-openshift-installer
+
+---
+- hosts: all
+  sudo: '{{ sudo | default("no") }}'
+  user: '{{ user | default("root") }}'
```

Comment:
I think doing this on the cli is a better approach. Not sure why it's here, other than an artifact of historical updates. I'll get a change in place for this one soon.